e2e/security/security.go

```diff
@@ -0,0 +1,218 @@
+// Copyright (c) 2019, Sylabs Inc. All rights reserved.
+// This software is licensed under a 3-clause BSD license. Please consult the
+// LICENSE.md file distributed with the sources of this project regarding your
+// rights to use or distribute this software.
+
+package security
+
+import (
+	"os"
+	"testing"
+
+	"github.com/sylabs/singularity/e2e/internal/e2e"
+	"github.com/sylabs/singularity/internal/pkg/buildcfg"
+)
+
+type ctx struct {
+	env e2e.TestEnv
+}
+
+// testSecurityUnpriv tests the security flag fuctionality for singularity exec without elevated privileges
+func (c *ctx) testSecurityUnpriv(t *testing.T) {
+	tests := []struct {
+		name       string
+		image      string
+		action     string
+		argv       []string
+		opts       []string
+		expectExit int
+	}{
+		// taget UID/GID
+		{
+			name:       "Set_uid",
+			image:      c.env.ImagePath,
+			action:     "exec",
+			argv:       []string{"id", "-u", "|", "grep", "99"},
```

Comment:
Pipe won't work here :

```suggestion
			argv:       []string{"id", "-u"},
```